docs/doxygen/overviews/eventhandling.h

```diff
@@ -349,6 +349,33 @@ MyFrame::MyFrame()
 }
 @endcode
 
+You can also bind functions that are members of any class provided that the
+class publicly derives from wxTrackable from somewhere down the line and that
+you have a pointer to that class object.
+For example:
+
+@code
+class MyFrame : public wxFrame
+{
+private:
+    void OnExit(wxCloseEvent& event);
+}
+
+void MyFrame::OnExit(wxCloseEvent& event)
+{
+    // ...
+}
+
+MyFrame::MyFrame()
+{
+    Bind(wxEVT_CLOSE_WINDOW, &MyFrame::OnExit, this);
+}
+@endcode
+
+You can also bind functions for classes that don't publicly derive from wxTrackable
+somewhere down the line, provided that your compiler supports C++11.
```

Comment:
Again, this seems wrong to me (and very confusing even if it were true, you can't say "you can only do this if that" only to add later "but you can also do this even if not that, sometimes").